### PR TITLE
Http full endpoint table hidden and csv

### PR DESCRIPTION
**Pull request by EmilyM1**

### CODE REVIEW

**Comment by EmilyM1** [comment]
Ready for code review

File: pkg/chargeback/http.go [hunk 1]
```diff
@@ -348,6 +348,65 @@ func (srv *server) getReport(logger log.FieldLogger, name, format string, useNew
 		srv.writeResults(logger, format, columns, results, w, r)
 	}
 }
+func (srv *server) returnWriteResultsCSV(logger log.FieldLogger, columns []api.PrestoTableColumn, results []map[string]interface{}, w http.ResponseWriter, r *http.Request) {
```
Comment (chancez) [inline pkg/chargeback/http.go]: can you rename this to `writeResultsAsCSV`? Otherwise LGTM.

**Comment by chancez** [comment]
Looks good, can you rebase? Ill merge after.

**Comment by EmilyM1** [comment]
Squashed, rebased and ready to merge

**Comment by chancez** [comment]
LGTM

**Pull request merged**